### PR TITLE
fozzie-components@v5.9.0 - Automatic package publishing to npm

### PR DESCRIPTION
**Pull request by siggerzz**
- Add new `publish-to-npm` workflow in `.circleci/config.yml` to automatically publish packages to npm as part of `master` builds
- Update Storybook / Contributing guides around npm publishing best practice.
- Update Percy CLI dependency.
- Add `"tag" : "beta` property to `@justeat/f-searchbox` `package.json` to ensure the version is tagged correctly once published.
- Make `@justeat/storybook` a private package so it's not published to npm

### CODE REVIEW

File: packages/components/molecules/f-searchbox/package.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "@justeat/f-searchbox",
   "description": "Fozzie Searchbox – Just Eat Takeaway Global Searchbox",
+  "tag": "beta",
```
Comment (siggerzz) [inline packages/components/molecules/f-searchbox/package.json:4]: By adding this property to beta packages, we ensure that the package will be tagged correctly on the registry, and won't override the `latest` release

File: packages/storybook/package.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
   "name": "@justeat/storybook",
+  "private":"true",
```
Comment (siggerzz) [inline packages/storybook/package.json:3]: Making this private as it's for monorepo use only, so there's no need to publish it!

File: .circleci/config.yml [hunk 3]
```diff
@@ -532,6 +543,16 @@ workflows:
               ignore: ['gh-pages']
           requires:
             - unit-tests-<< matrix.component_type >>
+      - publish-to-npm:
```
Comment (siggerzz) [inline .circleci/config.yml:546]: Ensure packages are only published on master once all builds are successful.

**Review by simonsandersje — APPROVED** [review]

**Review by JoshuaNg2332 — APPROVED** [review]

File: .circleci/config.yml (hunk 3 above)
Comment (ashleynolan) [reply .circleci/config.yml:546]: Out of interest, what happens if the build does fail? Does it just mean we'd have to run the publish manually or will it publish next time there's a green build?

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -73,26 +73,8 @@ If you are making changes that do not need a version bump in any part of the rep
...
-2. Update the components version in its `package.json` to be `3.0.0-beta.0`. Subsequent v3 beta releases will increment this integer suffix `-beta.x`
-3. Once a PR has been merged into the v3 release branch, the package should be published using the following command: `npm publish --tag beta`.
-   This publishes the package using the tag `beta` ensuring that it doesn't get tagged as the `latest` version of the package.
-4. When the package is ready for full release as v3, the `package.json` of the component should be updated to `3.0.0` in its `package.json` and a PR from the v3 release branch should be made back into the `main`/`master` branch. When this has been merged, the component can be published as normal using `npm publish`
+When working with beta package versions, you must ensure the `tag: "beta"` property is present in the package's `package.json` file **before** merging to master. Failure to do this will result in the `latest` version being overwritten on the registry.
```
Comment (ashleynolan) [inline CONTRIBUTING.md:80]: Worth having a chat about this flow. I'd recommend keeping beta releases manual and in a separate branch, mainly as there may also be ongoing dev happening on master concurrently which will need versioned releases.

Comment (ashleynolan) [reply CONTRIBUTING.md:80]: Also as a sidenote – I think searchbox is probably the edge case for how we usually develop beta releases (in that the original package is in another repo) so having the option to publish beta's like you've mentioned is still good, but not the typical use-case I would say.

File: stories/getting-started/contributing-guide.stories.mdx [hunk 5]
```diff
@@ -49,31 +52,11 @@ If you have any questions about how to implement any of these testing standards,
...
-2. Update the components version in its `package.json` to be `3.0.0-beta.0`. Subsequent v3 beta releases will increment this integer suffix `-beta.x`
-3. Once a PR has been merged into the v3 release branch, the package should be published using the following command: `npm publish --tag beta`.
-   This publishes the package using the tag `beta` ensuring that it doesn't get tagged as the `latest` version of the package.
-4. When the package is ready for full release as v3, the `package.json` of the component should be updated to `3.0.0` in its `package.json` and a PR from the v3 release branch should be made back into the `main`/`master` branch. When this has been merged, the component can be published as normal using `npm publish`
-
+When working with beta package versions, you must ensure the `tag: "beta"` property is present in the package's `package.json` file **before** merging to master. Failure to do this will result in the `latest` version being overwritten on the registry.
```
Comment (ashleynolan) [inline stories/getting-started/contributing-guide.stories.mdx:59]: Same here as above

**Pull request merged**